scripts/package-build/kea/package.toml

```diff
@@ -2,3 +2,4 @@
 name = "isc-kea"
 commit_id = "debian/2.4.1-3"
 scm_url = "https://salsa.debian.org/debian/isc-kea"
+build_cmd = "dpkg-buildpackage -uc -us -tc -F -j 1"
```

Comment:
Build fails
Did you check it in `vyos-build` container?
```
(Reading database ... 67171 files and directories currently installed.)
Preparing to unpack isc-kea-build-deps_2.4.1-3_all.deb ...
Unpacking isc-kea-build-deps (2.4.1-3) over (2.4.1-3) ...
Setting up isc-kea-build-deps (2.4.1-3) ...
dpkg-buildpackage: error: unknown option or argument 1

Use --help for program usage information.
Command 'dpkg-buildpackage -uc -us -tc -F -j 1' returned non-zero exit status 2.
I: Source packages build failed, ignoring - building binaries only
dpkg-buildpackage: error: unknown option or argument 1

Use --help for program usage information.
Failed to build package isc-kea: Command 'dpkg-buildpackage -uc -us -tc -F -j 1' returned non-zero exit status 2.
I: Cleaned up build dependency packages
vyos_bld@ce1d3ca789eb:/vyos/work/tmp/tmp-aws/vyos-build/scripts/package-build/kea$ 

```
With this change:
```
vyos_bld@ce1d3ca789eb:/vyos/work/tmp/tmp-aws/vyos-build/scripts/package-build/kea$ git diff .
diff --git a/scripts/package-build/kea/package.toml b/scripts/package-build/kea/package.toml
index 0bfce21e..1b06714b 100644
--- a/scripts/package-build/kea/package.toml
+++ b/scripts/package-build/kea/package.toml
@@ -2,3 +2,4 @@
 name = "isc-kea"
 commit_id = "debian/2.4.1-3"
 scm_url = "https://salsa.debian.org/debian/isc-kea"
+build_cmd = "dpkg-buildpackage -uc -us -tc -F -j 1"



```

Reply:
Minor typo, just needs the space removed between the flag and the value. Will push a fix now.

Reply:
The question the same.
Did you check it in container? It still fails

Reply:
There were a number of issues, but I've finally got it building in the container. Please check.

Reply:
build in container without any changes work fine 
What exactly you are trying to fix?

Reply:
The issue is thoroughly explained in the Phabricator task, I'm not quite sure how to explain it any more clearly. Builds don't always fail as-is, but they do fail sometimes because of the race condition. There is also a warning when building Kea to avoid using parallelism when compiling.